source/utils/HexagonMeshTools.h

```diff
@@ -0,0 +1,42 @@
+// ----------------------------------------------------------------------------
```

Comment:
On the line of simplifying the code and not using classes when they are not needed, I think these could be two functions. That would be similar to what is done in `IOUtils`.

Reply:
Done